### PR TITLE
Introduce 'View ratchetting', cleanup/improve vestibular motion

### LINKED ISSUE #495: Unset Setting has no Default
need to fix wrapper so this doesn't spam log.... either by setting to our default values or by just not writing to log.

### PR DESCRIPTION
**Pull request by feilen**
- Vestibular motion will now only activate if the user is moving within +-70 degrees of forward (according to their HMD's yaw), instead of activating on any movement, preventing the environment moving if you're somewhat swaying side to side. If desired, the margin could be made configurable.
- Introduce 'View ratchetting'. This acts somewhat like Vestibular Motion in that it gently nudges you away from facing the nearest wall, generally encouraging you to end up towards the center of your playspace. Instead of relying on positional movement, it works by magnifying any favorable head turning, by default by 5%. Originally I tried also minifying movements towards walls, but this ended up being the most comfortable (never requiring you to move your head extra, but sometimes allowing you to move it a little less). Needs GUI configurability, but settings values are implemented.
- Cleanup vestibular motion source somewhat.

ykeara:
somewhat cludge fix but it will work for now
fixes #495

fixed direct text input going to 3% for brightness slider [vs 10% from slider]

added UI for ratchet.

### CODE REVIEW

**Comment by feilen** [comment]
Fixed drift, just needs GUI now

**Review by ykeara — APPROVED** [review]
lg2m based on limited testing appears to work as long as algo is correct.

File: src/tabcontrollers/RotationTabController.cpp [hunk 1]
```diff
@@ -158,37 +163,99 @@ void RotationTabController::doVestibularMotion(
...
+            // Magnify that change
+            double delta_degrees
+                = reduceAngle<>(
+                      hmdToWallYaw - m_ratchettingLastHmdRotation, -M_PI, M_PI )
+                  * viewRatchettingPercent();
+
+            parent->m_moveCenterTabController.setRotation(
+                parent->m_moveCenterTabController.rotation()
+                + static_cast<int>( delta_degrees * k_radiansToCentidegrees ) );
+        } while ( false );
```
Comment (KCAFTS) [inline src/tabcontrollers/RotationTabController.cpp:224]: Why is this in a `do {} while (false)`?

**Review by username223 — APPROVED** [review]
Not tested in VR.

**Pull request merged**